cmake/onnxruntime_providers.cmake

```diff
@@ -794,7 +802,7 @@ if (onnxruntime_USE_OPENVINO)
   endif()
 
   source_group(TREE ${ONNXRUNTIME_ROOT}/core FILES ${onnxruntime_providers_openvino_cc_srcs})
-  onnxruntime_add_shared_library_module(onnxruntime_providers_openvino ${onnxruntime_providers_openvino_cc_srcs})
+  onnxruntime_add_shared_library_module(onnxruntime_providers_openvino ${onnxruntime_providers_openvino_cc_srcs} "${ONNXRUNTIME_ROOT}/core/dll/onnxruntime.rc")
```

Comment:
You can't say Microsoft is the author of the OpenVino EP DLL. Unless it meets Microsoft compliance requirements, it can't be published under the name of Microsoft. For example,  at very minimum Microsoft's legal team needs to know if this DLL contains any GPL code.

Reply:
I guess if it is breaking a security code we could create a new rc document for it. @jywu-msft can comment

Reply:
> > > This changes is required to ensure that we are able to host signed binaries in public domain
> > 
> > 
> > I have concerns on this. You can't publish binaries under the name of Microsoft **and** let Microsoft sign the binaries, when Microsoft's security and compliance team doesn't know anything about the binaries. If you do both of the things with your company's identity, then I won't have any concern because it is your business.
> 
> Microsoft is signing the onnxruntime.dll and onnxruntime_providers_shared.dll and Intel is signing the onnxruntime_providers_openvino.dll... that is the current process... It is the one we are following with MSFT and communicated to George too...

will follow up about this offline.

Reply:
> I guess if it is breaking a security code we could create a new rc document for it. @jywu-msft can comment

this part seems like a problem then. it shouldn't be marked as created by microsoft.